ui/src/index.js

```diff
@@ -1,21 +1,17 @@
-import { InvenioILSApp } from '@inveniosoftware/react-invenio-app-ils';
+import { history, InvenioILSApp } from '@inveniosoftware/react-invenio-app-ils';
 import React from 'react';
 import 'react-app-polyfill/ie11'; // For IE 11 support
 import ReactDOM from 'react-dom';
 import { OverridableContext } from 'react-overridable';
+import { Router } from 'react-router-dom';
 import 'semantic-ui-less/semantic.less';
-
-const CustomHome = ({ ...props }) => {
-  return <>CERN Library Catalogue</>;
-};
-
-const overriddenCmps = {
-  // 'Home.layout': CustomHome,
-};
+import { overriddenCmps } from './overridableMapping.js';
 
 ReactDOM.render(
-  <OverridableContext.Provider value={overriddenCmps}>
-    <InvenioILSApp />
-  </OverridableContext.Provider>,
+  <Router history={history}>
```

Comment:
explanation: it had to be added here, because the `<Link>` components were used out of router context, @ntarocco, @zzacharo do you see any possible trouble with this?

Reply:
The Router is already inside, then I would move it in ILS from the app to the `App.js`, to avoid to have it twice...